test/behavior/container_circular_dependency.zig

```diff
@@ -0,0 +1,41 @@
+const std = @import("std");
```

Comment:
Generally, try to avoid making overly specific files in `test/behavior/`. These would be better suited to `union.zig` and `struct.zig` respectively.

Also worth noting: this file isn't actually included in `test/behavior.zig` currently!

Reply:
Moved these into struct.zig / union.zig. I had originally made them their own file because they had to be in the root namespace to compile (otherwise `StructOuter` is an undefined identifier) and I didn't want to pollute the root namespace.